Cargo.toml

```diff
@@ -61,6 +61,12 @@ __tls = []
 # When enabled, disable using the cached SYS_PROXIES.
 __internal_proxy_sys_no_cache = []
 
+# feature to prevent building examples in wasm-builds
+# There will be a better way to do this issue #1197 is fixed
+# https://github.com/rust-lang/cargo/issues/1197
+# To build all examples, use cargo build --examples --features __examples
```

Comment:
Is there another way we can do this? I think requiring that people remember to enable this feature to see the examples work is too big a speed bump.

Reply:
fixed